### ISSUE TITLE
Fedora 40 install has missing GUI elements

### ISSUE THREAD

**Issue by ctheship** [equeim/tremotesf2#494]
Followed Copr install instructions for install.
```
sudo dnf copr enable equeim/tremotesf
sudo dnf install tremotesf

Return-Code    : Success
Releasever     : 40
Command Line   : install tremotesf
Comment        : 
Packages Altered:
    Install tremotesf-2.6.3-1.fc40.x86_64          @copr:copr.fedorainfracloud.org:equeim:tremotesf
    Install kde-filesystem-5-3.fc40.x86_64         @fedora
    Install kf6-filesystem-6.1.0-1.fc40.x86_64     @updates
    Install kf6-kwidgetsaddons-6.1.0-1.fc40.x86_64 @updates
    Install kf6-kwindowsystem-6.1.0-2.fc40.x86_64  @updates
```
Many elements are missing from the GUI and usability is reduced. Dark mode is not available.
![image](https://github.com/equeim/tremotesf2/assets/57539926/a77a977b-6eae-4c28-a2a1-7402d85ce688)

**Comment by equeim** [comment]
Dark mode thing is a Qt bug that will be properly resolved only in Qt 6.8 (I'm not sure they will backport it to 6.7).
For now you can add following snippet in `~/.config/gtk-3.0/settings.ini` :
```
[Settings]
gtk-application-prefer-dark-theme = true
```

Besides Qt it will also affect GTK 3 apps too.

**Comment by equeim** [comment]
Missing icons are fixed in 2.7.0

Dark theme is a bug in Qt
